frontend/src/main/webapp/static/js/service/poll_service.js

```diff
@@ -19,6 +19,8 @@ angular.module('myApp').factory('PollService', ['$http', '$q', function($http, $
             .then(
             function (response) {
                 deferred.resolve(response.data);
+				console.log(response)
```

Comment:
replace all instances of console.log by injecting $log, into your AngularJs component. and using $log.debug("") here.